xmtp/src/storage/encrypted_store/mod.rs

```diff
@@ -348,6 +348,31 @@ impl EncryptedMessageStore {
         Ok(())
     }
 
+    pub fn get_refresh_job(
```

Comment:
Is this meant to be part of this PR? Don't see it called anywhere

Reply:
I added it when I thought I was going to have to remove the transaction callback. Can remove until we need it.